kasa/iot/iotstrip.py

```diff
@@ -372,7 +372,9 @@ def on_since(self) -> datetime | None:
         info = self._get_child_info()
         on_time = info["on_time"]
 
-        return datetime.now().replace(microsecond=0) - timedelta(seconds=on_time)
+        return datetime.now(timezone.utc).astimezone().replace(
+            microsecond=0
+        ) - timedelta(seconds=on_time)
```

Comment:
I think the strips should also have the time module (albeit not all of them have the dst/time difference?). In order to properly support this, we'd need to create a mapping from timezone index to create proper timestamps.

Reply:
https://github.com/GadgetReactor/pyHS100/issues/53#issuecomment-296875129 ref, it's unknown if those are correct though.

Reply:
Yes I thought about addressing that but then realised there are complications with the `timezone` property and decided it was too much to risk for this release. Better to keep it more or less as is for now.

Reply:
What sort of complications are you seeing with that? I can take a look into this if I manage over the weekend. I doubt this information has been used that much, but as we are now going to expose the on_since, it'd be great to have it working properly :-)

Reply:
The complications were just me not knowing what the tz values would be on different devices, and worrying that I’d assume they were correctly based on some standard but then there would be breaking edge cases. I think you’ve addressed the some of the concerns in #980 so we could just go for it.